.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: ruby
+cache: bundler
```

Comment:
hm, apparently this is ineffective. https://docs.travis-ci.com/user/languages/ruby "Note that this feature [caching bundler] is currently only available for private projects."